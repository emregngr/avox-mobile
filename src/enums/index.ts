export const ENUMS = {
  API_TOKEN: 'api_token',
  IS_ONBOARDING_SEEN: 'is_onboarding_seen',
}
