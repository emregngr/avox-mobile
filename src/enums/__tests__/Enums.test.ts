import { ENUMS } from '@/enums'

describe('ENUMS Constants', () => {
  it('should have the correct and consistent values', () => {
    expect(ENUMS).toMatchSnapshot()
  })
})
