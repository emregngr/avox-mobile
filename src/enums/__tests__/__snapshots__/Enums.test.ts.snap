// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`ENUMS Constants should have the correct and consistent values 1`] = `
{
  "API_TOKEN": "api_token",
  "IS_ONBOARDING_SEEN": "is_onboarding_seen",
  "USER_ID": "user_id",
}
`;
